### PR TITLE
Check the description presence in the PR

### PR DESCRIPTION
**Pull request by rsvoboda**
Check the description presence in the PR

Discussion thread: https://groups.google.com/g/quarkus-dev/c/KHlsSiDhiEg/m/-rtxLgCDAgAJ

Examples:
 - https://github.com/rsvoboda/quarkus-github-app-playground/pull/5
 - https://github.com/rsvoboda/quarkus-github-app-playground/pull/6

Tests which had multi-line comment check were transformed to text blocks and line with description complain was added.
Other tests were not adjusted, relevant json files were updated not to touch test logic.

Please instruct me if you would prefer different approach to tests changes.

### CODE REVIEW

**Comment by rsvoboda** [comment]
@gsmet will you look into it. You mentioned that you wanted to do some docs related checks. Or hint me in the comment about the classes to look into

File: src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java [hunk 1]
```diff
@@ -132,4 +135,28 @@ private static boolean isUpperCaseException(String title) {
...
+        // Note: these rules will have to be adjusted depending on how it goes
+        // we don't want to annoy people fixing a typo or require a description for a one liner explained in the title
+
+        // if we have more than one commit, then it's meaningful
+        if (pullRequest.getCommits() > 1) {
+            return true;
+        }
+
+        // for one liner/two liners, let's be a little more lenient
+        if (pullRequest.getAdditions() <= 2 && pullRequest.getDeletions() <= 2) {
```
Comment (rsvoboda) [inline src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java:157]: Component version updates will go through this without notice. We had SmallRye Config updates like that and they were minor version bumps ...

**Review by rsvoboda — COMMENTED** [review]
In my local change I focused to be lenient just for docs changes
Even one-liners can be impactful 

My check was:
```java
    private static boolean smallDocsUpdate(GHPullRequest pullRequest) throws IOException {
        if (pullRequest.getChangedFiles() == 1) {
            
            PullRequestFilesMatcher prMatcher = new PullRequestFilesMatcher(pullRequest);
            boolean docsChange = prMatcher.changedFilesMatch(List.of("*.adoc", "*.md"));

            int changes = 0;
            if (pullRequest.listFiles() != null) {
                changes = pullRequest.listFiles().iterator().next().getChanges();
            }

            return docsChange && changes <= 2;
        }
        return false;
    }
```

File: src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java (hunk 1 above)
Comment (gsmet) [reply src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java:157]: Sorry but I really don't want to bug everyone for very small changes. It's going to be a pain and we will end up with non-detailed descriptions anyway.

So I tried to find a compromise that would be acceptable by the developers.

Comment (rsvoboda) [reply src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java:157]: In general I agree that we shouldn't put barrier for contributions.

But maybe we should be a bit stricter for changes in https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml.
I would mark any PR  that touches `bom/application/pom.xml` as meaningful pull request.

Comment (gsmet) [reply src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java:157]: I added a commit with that.

Ideally, I think we should deploy and gather feedback and improve from there.

Comment (gsmet) [reply src/main/java/io/quarkus/bot/CheckPullRequestEditorialRules.java:157]: @rsvoboda I let you check the last commit does what you wanted and then I would be in favor of merging and deploying.

**Review by rsvoboda — COMMENTED** [review]
https://github.com/quarkusio/quarkus-github-bot/pull/447/commits/f8f378379eb5e65b9e3016101256c67d53ca0aaa looks good, thank you for adding that!

**Comment by gsmet** [comment]
I'm on PTO tomorrow so I'll push it on Monday.

**Pull request merged**